neurodamus/connection_manager.py

```diff
@@ -840,13 +846,14 @@ def get_target_connections(self, src_target_name,
         if src_target and src_target.is_void() or dst_target.is_void():
             return
 
-        _, tgid_offset = self.get_population_offsets()
-        populations: List[ConnectionSet] = (conn_population,) if conn_population is not None \
+        tgid_offset = self.target_pop_offset
```

Comment:
Can we remove the func `get_population_offsets()` ?

Reply:
good point. That was a reminiscent from the hoc targets... 😅